src/garage/torch/algos/ppo.py

```diff
@@ -59,7 +59,9 @@ def __init__(self,
                  policy_ent_coeff=0.0,
                  use_softplus_entropy=False,
                  stop_entropy_gradient=False,
-                 entropy_method='no_entropy'):
+                 entropy_method='no_entropy',
+                 minibatch_size=None,
+                 optim_epochs=1):
```

Comment:
please use full words. also, some minibatch optimizers use early stopping, so i think a more general arg name is `max_optimization_epochs`

Reply:
fixed